cmd/vic-machine/create/create.go

```diff
@@ -114,18 +121,33 @@ func (c *Create) Flags() []cli.Flag {
 			Destination: &c.DisplayName,
 		},
 		cli.StringFlag{
-			Name:        "external-network",
-			Value:       "",
-			Usage:       "The external network (can see hub.docker.com)",
-			Destination: &c.ExternalNetworkName,
-		},
-		cli.StringFlag{
 			Name:        "management-network",
 			Value:       "",
 			Usage:       "The management network (can see target)",
 			Destination: &c.ManagementNetworkName,
 		},
 		cli.StringFlag{
+			Name:        "client-network",
+			Value:       "",
+			Usage:       "The client network (restricts DOCKER_API access to this network)",
+			Destination: &c.ClientNetworkName,
+		},
+		cli.StringSliceFlag{
+			Name:  "container-network",
+			Value: &c.containerNetworks,
+			Usage: "Networks that containers can use",
```

Comment:
If there's a Description field that should be used for the current string. The Usage should include the format specification and not make people experiment to get error output with that detail.